contrib/segmentio/kafka.go.v0/kafka_test.go

```diff
@@ -84,6 +84,7 @@ func TestReadMessageFunctional(t *testing.T) {
 	assert.Equal(t, 0, s0.Tag("partition"))
 	assert.Equal(t, "segmentio/kafka.go.v0", s0.Tag(ext.Component))
 	assert.Equal(t, ext.SpanKindProducer, s0.Tag(ext.SpanKind))
+	assert.Equal(t, "kaka", s0.Tag(ext.MessagingSystem))
```

Comment:
```suggestion
	assert.Equal(t, "kafka", s0.Tag(ext.MessagingSystem))
```

Reply:
Already Fixed!

Reply:
Lol as soon as I hit submit on the comment I saw that I needed to refresh xP